anomalib/deploy/export.py

```diff
@@ -17,7 +17,7 @@
 from anomalib.models.components import AnomalyModule
 
 
-class ExportMode(str, Enum):
+class OutputFormat(str, Enum):
```

Comment:
Is there any motivation or design discussions for this?

Reply:
I have the feeling that we should have `Export` in the naming. How about `ExportFormat`?

Reply:
I feel `format` is more descriptive than `mode`, because the only thing this parameter changes is the final format of the model files that the exporter produces at the output. I'm fine with `ExportFormat`.